packages/big-design/src/components/Select/Select.tsx

```diff
@@ -74,6 +74,10 @@ export class Select extends React.PureComponent<SelectProps, SelectState> {
     if (prevProps.value !== value) {
       this.updatedSelectedItem();
     }
+
+    if (prevProps.children !== children) {
```

Comment:
Should we be returning early when calling `updatedSelectedItem`? 

Seems like we could be calling it multiple times.

Reply:
☝️That makes sense. Should've added it before.

Reply:
I'll add that in